### PR TITLE
Adds start button to initialize

### PR DESCRIPTION
**Pull request by dom-tucker**
This PR address issue #1

### CODE REVIEW

**Review by nwbauer — CHANGES_REQUESTED** [review]
There was a request on the issue #1 to remove the initial map. Can you add those changes as well?

**Comment by dom-tucker** [comment]
Added commit to comment out initial table data. Future plan to update graphics.

**Review by nwbauer — APPROVED** [review]

**Comment by nwbauer** [comment]
Cool.  Thanks!

File: src/index.html [hunk 1]
```diff
@@ -4,9 +4,13 @@
     <link rel="stylesheet" href="./styles-manhattan.css">
     <title> Manhattan Project </title>
     <script type="application/javascript" src="./main.js"></script>
+    <button type="button" onclick="main()">Start</button>
+
   </head>
   <body>
+
     <table class="map">
+      <!--
```
Comment (nwbauer) [inline src/index.html:13]: Reminder: Need to replace graphics so we can remove this.

**Pull request merged**